astropy/cosmology/tests/test_flrw.py

```diff
@@ -1221,12 +1283,22 @@ def test_clone_change_param(self, cosmo):
             else:
                 assert u.allclose(v, getattr(cosmo, n), atol=1e-4 * getattr(v, "unit", 1))
 
+    # @pytest.mark.parametrize("z", valid_zs)
```

Comment:
As above. Maybe be explicit if this is meant to be helpful for future devs

Reply:
I added the quick comment ``  # TODO! recompute comparisons below``